src/main/java/org/ld4l/bib2lod/managers/SimpleManager.java

```diff
@@ -63,19 +78,41 @@ private static void setup(String[] args) {
         configuration = new ConfigurationOverrider(commandLine)
                 .override(configuration);
         configuration = new AttributeCascader().cascade(configuration);
-
-        Bib2LodObjectFactory.setFactoryInstance(
-                new DefaultBib2LodObjectFactory(configuration));
-
+        setupObjectFactory(configuration);
+    }
+    
+    /**
+     * Reads the config file to set up application without any (command line) adjustments. 
+     * 
+     * @param jsonConfigFilePath - Path to the JSON configuration file.
+     */
+    public SimpleManager(String jsonConfigFilePath) {
+    	super();
+    	InputStream jsonInput = readConfigFile(jsonConfigFilePath);
```

Comment:
Where is jsonInput closed? Either a try-finally or a try-with-resources would be good.